### PR TITLE
Import currency symbols from CLDR Unicode github repo - VPN-4306

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

**Review by flodolo — COMMENTED** [review]
Is there a specific reason to write _ad hoc_ code on top of CLDR, and not using qt libraries directly?
https://doc.qt.io/qt-5/qml-qtqml-number.html#toLocaleCurrencyString-method

**Comment by bakulf** [comment]
We use this method but we need to pass the symbol.

**Comment by bakulf** [comment]
Note that we do this only as a fallback when the currency symbol of the current locale does not match with the requested currency.

File: src/apps/vpn/translations/i18n
Comment (flodolo) [inline src/apps/vpn/translations/i18n:1]: Not sure this change is expected

**Review by flodolo — APPROVED** [review]

**Review by strseb — APPROVED** [review]

**Pull request merged**